app/main.scss

```diff
@@ -8,3 +8,10 @@
 .app-width-container {
   @include govuk-width-container(1280px);
 }
+
+.app-section-row {
+  display: flex;
+  @include govuk-media-query($until: mobile) {
```

Comment:
I think you possibly mean `$until: tablet`. At the moment this is only applying for screens smaller than mobile (which is why on mobile devices I'm still getting two columns).